modules/end-user-guide/partials/proc_creating-a-workspace-by-importing-the-source-code-of-a-project.adoc

```diff
@@ -3,49 +3,62 @@
 :parent-context-of-creating-a-workspace-by-importing-the-source-code-of-a-project: {context}
 
 [id="creating-a-workspace-by-importing-the-source-code-of-a-project_{context}"]
-= Creating a workspace by importing the source code of a project
+= Creating a workspace with your project's source code
```

Comment:
Reasons for using _with_ and _your_:

_with_:
1) The end result in all four sections in this file is a workspace _with_ the user's source code in it, so this heading is equally applicable to and valid for all four sections and describes their common result.
2) The end result (a workspace with your project's source code in it) has the primary value to the user/reader, not the procedure of importing, so when the user is scanning through the docs, the user is looking to find a section that will enable the user to get the end result that the user has in mind (= a workspace with the user's source code). This means better usability for both the software and the docs and better match with the user story.

_your_ = To the user/reader who reads "your", it means what the user/reader owns and brings, not something that is provided by the software. What the user/reader owns and brings is the reason why the user/reader decided to use this software. Also, the distinction between the user's and the software's is worth emphasizing here to for better usability and better match with the user story.

Reply:
Robert, your suggestion is the best in my opinion:
**Importing the source code of a project into a workspace** covers all the sections.
Should I make this heading change then?